pkgs/servers/web-apps/photoprism/generate-dependencies.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env nix-shell
+#! nix-shell -i bash -p nodePackages.node2nix
```

Comment:
This currently needs a development build of https://github.com/svanderburg/node2nix because it depends on svanderburg/node2nix#199

Reply:
The PR you mentioned was merged. Waiting for a release.

Reply:
There's the general question, if we want to keep the node2nix approach or rather use https://github.com/tweag/npmlock2nix
I've briefly tried to make the change, but failed.